src/Common/src/System/Security/Cryptography/Asn1Reader/AsnCharacterStringEncodings.cs

```diff
@@ -65,7 +65,7 @@ public override int GetByteCount(string s)
         }
 
         public
-#if netcoreapp || uap || NETCOREAPP
+#if netcoreapp || uap || NETCOREAPP || netstandard2_1
```

Comment:
What is the difference in between lowercase `netcoreapp` and uppercase `NETCOREAPP`?

Reply:
This source is shared in corefx and corefxlab.  Corefx uses `netcoreapp`, corefxlab uses `NETCOREAPP`.

Reply:
Ok, that is why I couldn't find any instance of `NETCOREAPP`. Thanks.